### DISCUSSION TITLE
Notifications/Webhooks

### DISCUSSION THREAD

**Discussion by kieron** [strapi/strapi#6666] [Legacy Topics]
Hi all.

First thing, I'm trying to add a discord webhook to my Strapi, I have set the events to `Entry` and then applied, create, edit and delete. In headers, I have applied a `Content-Type` of `application/json` as normal, but if I trigger, I get a 400 error, of `{"message": "Cannot send an empty message", "code": 50006}`

Understandably, I thought OK maybe the test doesn't send any content, however, its also not firing on creation/edit/deletion, that I do via the API routes, so question is, does a webhook not send any content when it supposedly fire?

---

Second question, how can I edit this content, I am envisioning that the webhook can be sent over with a link of the content within Strapi, so that I can click and be taken straight to the content item, or is this more suited to email notifications, can Strapi have those?

Thanks!

**Comment by ricardoboss** [comment]
I am also interested in a solution for this. Can we edit the content?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by ricardoboss** [reply]
Thanks for the reply. Will post the questions there!
